### PR TITLE
NEWS: Update German translation for recent adibou2 additions

### PR DESCRIPTION
**Pull request by CrystalBen**
Some German Translations
Changed Adi Junior to Addy Junior
Changed Read/Count 4 & 5 to Lesen/Rechnen 4 & 5
Changed Read/Count 6 & 7 to Lesen/Rechnen 6 & 7

### CODE REVIEW

File: doc/de/NEUES.md [hunk 1]
```diff
@@ -61,7 +61,7 @@ Umfangreichere Informationen über die Änderungen des aktuellen Codes findest D
    - Unterstützung für Obsidian hinzugefügt.
    - Unterstützung für Added support for Pink Panther: Passport to Peril hinzugefügt.
    - Unterstützung für Pink Panther: Hokus Pokus Pink hinzugefügt.
-   - Unterstützung für Adibou 2 (Adi Junior) "Environment", "Read/Count 4 & 5" und "Read/Count 6 & 7" hinzugefügt.
+   - Unterstützung für Adibou 2 (Addy Junior) "Environment", "Lesen/Rechnen 4 & 5" und "Lesen/Rechnen 6 & 7" hinzugefügt.
```
Comment (lotharsm) [inline doc/de/NEUES.md:64]: According to the title screen, we should leave it as "Adi", at least that's how the version I have shows it. I agree with "Lesen/Rechnen" though.
scummvm-adibou2-win-de-00001

**Comment by CrystalBen** [comment]
i applied the changes

**Comment by CrystalBen** [comment]
is this a picture of a german version?

**Comment by lotharsm** [comment]
> is this a picture of a german version?

Yes, this screenshot is taken with the following version:
```
	{
		"adibou2",
		"ADI Jr. (Packard Bell OEM)",
		AD_ENTRY1s("intro.stk", "80588ad3b5510bb44d3f40d6b07b81e7", 956328),
		DE_DEU,
		kPlatformWindows,
		ADGF_NO_FLAGS,
		GUIO0()
	},
	kGameTypeAdibou2,
	kFeatures640x480,
	0, 0, 0
},
```

**Comment by CrystalBen** [comment]
ah okay

**Comment by lotharsm** [comment]
Do you want to add the changes to your PR/commits or do you want me to quickly apply the changes manually? :-)

**Comment by CrystalBen** [comment]
can you please apply these changes for me?

**Pull request closed without merging**